ui/index.html

```diff
@@ -7,7 +7,7 @@
     <title>OpenNMS Web Console</title>
   </head>
   <body>
-    <div id="app"></div>
+    <div id="app" class="feather-styles"></div>
```

Comment:
Is use of this here documented anywhere?

Reply:
Where do i need to add documentation for this change?

Reply:
I mean is this class itself in the feather documentation? I took a quick look and didn't see it so I'm wondering if there are examples. I noticed it adds underlines to the links on the sidebar, which I don't think we should be doing.